lib/rufo/settings.rb

```diff
@@ -4,17 +4,26 @@ module Rufo::Settings
     align_case_when: [false, true],
     align_chained_calls: [false, true],
     trailing_commas: [true, false],
+    print_width: (1..Float::INFINITY)
+  }
+
+  DEFAULT_VALUES = {
+    print_width: 80
   }
 
   attr_accessor *OPTIONS.keys
 
   def init_settings(options)
     OPTIONS.each do |name, valid_options|
-      default = valid_options.first
+      if DEFAULT_VALUES.has_key?(name)
+        default = DEFAULT_VALUES[name]
+      else
+        default = valid_options.first
+      end
       value = options.fetch(name, default)
       unless valid_options.include?(value)
         $stderr.puts "Invalid value for #{name}: #{value.inspect}. Valid " \
-                     "values are: #{valid_options.map(&:inspect).join(', ')}"
+                      "values are: #{valid_options.map(&:inspect).join(', ')}"
```

Comment:
Why did a space get added here? Was it added manually, or did rufo do that in the formatting pass?

Reply:
Um not sure to be honest. I will run the formatter over it once the tests are passing. Hopefully that will remove it.